app/javascript/packs/document-capture.jsx

```diff
@@ -34,6 +34,7 @@ loadPolyfills(['fetch']).then(() => {
       <UploadContextProvider
         endpoint={appRoot.getAttribute('data-endpoint')}
         csrf={getMetaContent('csrf-token')}
+        sessionUUID={appRoot.getAttribute('data-document-capture-session-uuid')}
```

Comment:
I would maybe call this something like `formFields` or something or can we just add a hidden input tag inside here that gets submitted? That way we can also pass in the harcoded `locale` param without having to plumb each key all the way through

```jsx
formFields={{
  document_capture_session_uuid: appRoot.getAttribute('data-document-capture-session-uuid'),
  locale: getLocale()
}}
```

Reply:
Paired with Andrew and wired this up